src/main/java/no/nav/pto/veilarbportefolje/arenapakafka/ytelser/YtelsesService.java

```diff
@@ -212,4 +193,30 @@ private boolean erGammelMelding(YtelsesDTO kafkaMelding, YtelsesInnhold innhold)
         }
         return false;
     }
+
+    private void oppdaterAktivYtelse(AktorId aktorId, YtelseDAO ytelse) {
+        Brukerdata ytelsesTilstand = new Brukerdata()
+                .setAktoerid(aktorId.get());
+        if (ytelse == null) {
+            ytelsesStatusRepositoryV2.upsertYtelse(ytelsesTilstand);
```

Comment:
Er dette i praksis å slette (`null` for alle felter)? Trenger man beholde raden?

Reply:
Oj godt spørsmål, det kan hende at det er riktig ja 👍  
I oracle så delte ytelses radene samme tabell som noen av aktivitet data (feks. har utløpt aktivitet)